src/main/java/bean/Orders.java

```diff
@@ -20,12 +20,9 @@ public class Orders {
 
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "users_id")
+    @JoinColumn(name = "id")
     private Users users;
 
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "status_orders_id")
-    private StatusOrder statusOrder;
```

Comment:
Why did you remove it?

Reply:
Because have problem with idea, has exception!